bin/tessel-2.js

```diff
@@ -186,12 +189,24 @@ parser.command('init')
 
 parser.command('wifi')
   .callback(function(opts) {
-    //TODO: Refactor switch case into controller.wifi
+    // TODO: Refactor switch case into controller.wifi
     if (opts.list) {
       controller.printAvailableNetworks(opts)
         .then(closeSuccessfulCommand, closeFailedCommand);
-    } else if (opts.ssid && opts.password) {
-      controller.connectToNetwork(opts)
+    } else if (opts.ssid || opts.password) {
+      if (opts.ssid && opts.password) {
```

Comment:
@MrNice On second thought, I take this back. Once we make this CLI more usable from other Node programs, it would be great if we could export functions so having this extra check doesn't hurt. In fact, because this check is removed they are causing [this test](https://github.com/tessel/t2-cli/pull/282/files#diff-7275035954a27dab20f24918ba5bea6aR146) and [this test](https://github.com/tessel/t2-cli/pull/282/files#diff-7275035954a27dab20f24918ba5bea6aR160) to fail right now.